aiida_quantumespresso/parsers/parse_raw/pw.py

```diff
@@ -270,10 +270,15 @@ def parse_stdout(stdout, input_parameters, parser_options=None, parsed_xml=None)
     relax_steps = [i.split('\n') for i in relax_steps]
 
     # now I create a bunch of arrays for every step.
-    for data_step in relax_steps:
 
+    # initialize the scf_index counter for the next cycle
+    scf_index = 0
+    for data_step in relax_steps:
         trajectory_frame = {}
-
+        try:
+            trajectory_data['scf_accuracy_index'].append(scf_index)
```

Comment:
There is a useful paradigm in python for this situation:
```
trajectory_data.setdefault('scf_accuracy_index', []).append(scf_index)
```
This will append the value to the list if it exists, or will create the list first if it does not yet exist

Reply:
@sphuber That was actually my first idea, but I opted to keep consistency with the rest of the try...- except sections of the parser. I can go through those too in other places and change it if you prefer

Reply:
I see. Consistency is always good. But let's change them all for the more idiomatic version of `setdefault`.